webapp/src/DOMJudgeBundle/Resources/views/jury/config.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends "@DOMJudge/jury/base.html.twig" %}
+{% import _self as self %}
+{% import "@DOMJudge/jury/jury_macros.twig" as macros %}
+
+{% block title %}Configuration - {{ parent() }}{% endblock %}
+
+{% block extrahead %}
+    {{ parent() }}
+    {{ macros.table_extrahead() }}
+{% endblock %}
+
+{% block content %}
+    <h1>Configuration</h1>
+    <div class="alert alert-danger" role="alert">View only page, edit values <a href="config.php">here</a></div>
+    <form>
+    {% for category in options %}
+        <div class="card">
```

Comment:
Indenting looks a bit weird IMHO

Reply:
Is there an easy way to autoformat this?

Reply:
Code menu in PHPStorm and then `Reformat code` should do the trick.

Reply:
Done.